### PR TITLE
[StringBundler] LPS-84182 Apply Petra-String StringBundler to method argument and return value part1

### PR DESCRIPTION
**Pull request by ChiZelin**

### CODE REVIEW

**Comment by ChiZelin** [comment]
ci:test:sf

**Comment by ChiZelin** [comment]
ci:test

**Comment by ChiZelin** [comment]
ci:test

**Comment by dantewang** [comment]
Fetched

File: portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/aui/TagInvocationData.java [hunk 1]
```diff
@@ -30,6 +30,18 @@
  */
 public class TagInvocationData implements Serializable {
 
+	/**
+	 * @deprecated As of Athanasius (7.3.x), replaced by {@link
+	 *			 #TagInvocationData(StringBundler, String)}
+	 */
+	@Deprecated
+	public TagInvocationData(
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/aui/TagInvocationData.java]: @ChiZelin This constructor has a usage in `ScriptData`. I can see that you marked `ScriptData.append` as blocked by SB usages in return values. But the usage of this constructor is inside a inner private class. Could you check if you can at least replace that usage? We should avoid keeping usage to deprecated methods as much as we can.

File: portal-kernel/src/com/liferay/portal/kernel/search/DocumentImpl.java [hunk 2]
```diff
@@ -1057,6 +1057,22 @@ protected void setSortableTextFields(Set<String> sortableTextFields) {
...
+	/**
+	 * @deprecated As of Athanasius (7.3.x), replaced by {@link
+	 *			 #toString(StringBundler, Collection)}
+	 */
+	@Deprecated
+	protected void toString(
+		com.liferay.portal.kernel.util.StringBundler sb,
+		Collection<Field> fields) {
+
+		StringBundler petraSB = new StringBundler();
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/portal/kernel/search/DocumentImpl.java]: @ChiZelin Just to confirm, this method is meant to populate strings into the SB, right?

Comment (ChiZelin) [reply portal-kernel/src/com/liferay/portal/kernel/search/DocumentImpl.java]: Yes

File: portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/aui/TagInvocationData.java (hunk 1 above)
Comment (ChiZelin) [reply portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/aui/TagInvocationData.java]: Yes, I will recheck it.

Comment (ChiZelin) [reply portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/aui/TagInvocationData.java]: @dantewang The actual argument of usages of ```com.liferay.portal.kernel.servlet.taglib.aui.ScriptData#append(java.lang.String, com.liferay.portal.kernel.util.StringBundler, java.lang.String, com.liferay.portal.kernel.servlet.taglib.aui.ScriptData.ModulesType)``` is returned by SB usages in return values. Maybe it's better to do this one after SB usages in return values. I will revert this commit.

**Comment by ChiZelin** [comment]
Resend, please see https://github.com/dantewang/liferay-portal/pull/1543

**Pull request closed without merging**